### PR TITLE
Mandarin AISHELL1

### PR DESCRIPTION
**Pull request by lifeiteng**
- [x] refactor TextTokenizer

### CODE REVIEW

File: valle/data/tokenizer.py [hunk 1]
```diff
@@ -29,6 +30,90 @@
...
+        self.punctuation_marks = punctuation_marks
+
+    def phonemize(
+        self, text: List[str], separator: Separator, strip=True, njobs=1
+    ) -> List[str]:
+        assert isinstance(text, List)
+        phonemized = []
+        for _text in text:
+            _text = re.sub(" +", " ", _text.strip())
+            _text = _text.replace(" ", separator.word)
```
Comment (zhaomingwork) [inline valle/data/tokenizer.py:61]: "此项 工作 还能 怎么 改进"=>'ci3_xiang4_gong1_zuo4_hai2_neng2_zen3_me5_gai3_jin4'. If  want to keep original <SPACE>, seems a problem. 
or use something like '%' to replace original SPACE =>'ci3_xiang4_%_gong1_zuo4_%_hai2_neng2_%_zen3_me5_%_gai3_jin4'.

Comment (lifeiteng) [reply valle/data/tokenizer.py:61]: > "此项 工作 还能 怎么 改进"=>'ci3_xiang4_gong1_zuo4_hai2_neng2_zen3_me5_gai3_jin4'. If want to keep original SPACE, seems a problem. or use something like '%' to replace original SPACE =>'ci3_xiang4_%_gong1_zuo4_%_hai2_neng2_%_zen3_me5_%_gai3_jin4'.

I think we should use different tokens to represent the internal and boundary of `WORD`:
`ci3-xiang4_gong1-zuo4 ...`

**Review by zhaomingwork — APPROVED** [review]

File: valle/data/tokenizer.py (hunk 1 above)
Comment (chenjiasheng) [reply valle/data/tokenizer.py:61]: What about commas(逗号) and periods(句号、问号、感叹号)?
These marks have explicit pause and emotion prompt for the synthesis.
AISHELL dataset may not contain them, but still we can add them for future compatibility.

Comment (zhaomingwork) [reply valle/data/tokenizer.py:61]: any suggestion for code? or you mean it is aishell1's problem.

**Pull request merged**